tests/sandbox/test_405_errors.py

```diff
@@ -0,0 +1,35 @@
+import requests
+import pytest
+from lib import Assertions, Generators
+from lib.constants.messages_paths import MESSAGES_ENDPOINT
+from lib.constants.message_batches_paths import MESSAGE_BATCHES_ENDPOINT
+
+INVALID_URI_METHOD_COMBINATIONS = [
+    (MESSAGE_BATCHES_ENDPOINT, "GET"),
```

Comment:
As the API supports `GET`, `POST`, `PUT`, `PATCH`, `DELETE` and `HEAD` we should check for these as part of the tests